### ISSUE TITLE
How to handle tab delimited csv file?

### ISSUE THREAD

**Issue by metawops** [microsoft/SandDance#261]
I have a csv file with the tab character als a delimiter.
I'm in VSCode and have the file open.
Extensions like Rainbow CSV have no problem detecting the columns.
But SandDance doesn't recognize my columns. It just sees one single column.
How can I tell SandDance (the VSCode extension) to use the tab character as a delimiter?

**Comment by danmarshall** [comment]
Hi @metawops, can you try changing the file extension to `tsv` to see if it works? Typically `.csv` files use a comma, and `.tsv` files use a tab as delimiters.

We could add a features to do better column detection, as well as handle semicolon and international number formats which use a comma as a decimal.

**Comment by metawops** [comment]
> Hi @metawops, can you try changing the file extension to `tsv` to see if it works? Typically `.csv` files use a comma, and `.tsv` files use a tab as delimiters.
>

That actually worked. However, Windows Explorer now doesn’t show the Excel document icon anymore because Excel apparently isn’t connected with `.tsv` files. 
Anyway, thanks for the hint!
 
> We could add features to do better column detection, as well as handle semicolon and international number formats which use a comma as a decimal.

I’d appreciate that. Always a hassle working with decimal points/commas in number, date formats, commas in strings that are not field separators etc.pp.

**Comment by Runekeon** [comment]
I agree working with nested json in colunms so using ~ for separator. The ~ never shows up in the data but almost any other character could.

**Comment by hammerandpick** [comment]
> Quick fix could be to make the column separator an editable setting ;-)
> 
> Need semicolon as separator, because comma is the decimal separator in german!

Same here, this problem is due to the switched "." and "," within numbers. I raised this as idea #606 before checking for this issue.

**Comment by verajosemanuel** [comment]
same here. Comma separator is for decimals in spanish, so unable to use SandDance in vscode

**Comment by ShinySky42** [comment]
Grave digging but the column separator needs to be an editable setting.
